dashboard/context_processors.py

```diff
@@ -35,6 +35,14 @@ def course_view_option(request):
 
     return {"course_view_option": course_view_option}
 
+def current_user_prefixed_course_id(request):
+    course_id = str(request.resolver_match.kwargs.get('course_id'))
+    if not course_id:
+        logger.info("Course ID could not be determined from request, attempting to look up for user {}".format(
```

Comment:
use of Python3's f-string to do logging?

https://realpython.com/python-f-strings/

Reply:
Switched to f-string. 

Note that the other logging in the file is done in the same format I used. Is this a change you'd want to apply to `.format()` uses in the project?